GetDKAN/dkan
# Update workbench_moderation, drafty, and dkan_sitewide_panels

connects #2578
Fix for 'View Draft' tab

Updates module versions and changes to panel features. Because we're using panels to display the dataset and resource pages, we need to use "node_content" to place the content in the panel rather than "entity_view," which will not allow us to see the draft version.

## How to reproduce

0. Enable dkan_workflow
1.  Go to an existing dataset, edit it, make changes and save as draft
2. Click the 'View Draft' tab
3. Confirm that you are viewing the published version rather than the draft

## QA Steps
- [ ] Enable dkan_workflow
- [ ]  Go to an existing dataset, edit it, make changes and save as draft
- [ ] Click the 'View Draft' tab
- [ ] Confirm that you are viewing the draft version
- [ ] Repeat steps on a resource

## Reminders
